scripts/seed_rag_freshbus.py

```
@@ -0,0 +1,452 @@
+        all_embs.extend(embs)
+        print(f"  Embedded {min(i + batch_size, len(texts))}/{len(texts)} chunks …")
+    return np.array(all_embs, dtype=np.float32)
+
+
+# ---------------------------------------------------------------------------
+# Main
+# ---------------------------------------------------------------------------
+
+
+async def main() -> None:
+    print(f"\n=== FreshBus RAG seed script ===")
```

Ruff (0.15.11)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)